hasura/graphql-engine
# console: display rows limit in permissions editor if set to zero (fix #4559)

Solves #4559 

### Description

Changed  `permissionsState.select.limit ? permissionsState.select.limit : '';`  to `exists(permissionsState.select.limit) ? permissionsState.select.limit : '';` so that we display `permissionsState.select.limit` if it's set to zero, and placeholder `''` only if it's `undefined` or `null`.


### Changelog


### Affected components


### Steps to test and verify

1. Go to the permissions tab.
0. Open permissions editor for `select` action.
0. Use the number picker or arrow keys to go down from _some value_ to zero. 

Current behavior: `0` is not displayed in the input.
Expected behavior: `0` is displayed in the input.
